ecommerce/frontend/src/components/messages/Conversations.less

```diff
@@ -0,0 +1,24 @@
+.conversations {
+    display: grid;
+    grid-template-columns: minmax(auto, 300px) 1fr;
+    overflow: auto;
+    width: 100%;
+    min-height: 500px;
+}
+
+.conversations-empty {
+    display: grid;
+    overflow: auto;
+    width: 100%;
+    min-height: 500px;
+    place-content: center;
+}
+
+.conversations-list {
```

Comment:
Layout is looking nice and clean. 
I think a gray vertical separator between the conversations list and messages could be nice. As the one in tabs pane on left.
![image](https://user-images.githubusercontent.com/44057640/104813491-90671200-581a-11eb-9c83-1ad723355edb.png)
Also, upload and send button at the bottom can have 1-2px gap

Reply:
👍🏼